BuildAndTest.proj

```diff
@@ -99,18 +99,29 @@
       <VsixExpInstallerArgs>/rootSuffix:RoslynDev /vsInstallDir:"$([System.IO.Path]::GetFullPath('$(MSBuildBinPath)\..\..\..'))"</VsixExpInstallerArgs>
     </PropertyGroup>
 
-    <Exec Condition="'$(TestVsi)' != 'true' AND '$(SkipCoreClr)' != 'true'" Command="&quot;$(CoreRunExe)&quot; $(CoreRunArgs)" />
+    <!-- Run CoreClr tests -->
+    <Exec Condition="'$(TestVsi)' != 'true' AND '$(SkipCoreClrTests)' != 'true'" 
```

Comment:
This build file now has both positive and negative ways of controlling tests:

- positive: TestVsi
- negative: SkipCoreClrTests, SkipDesktopTests

We should unify the properties here.  Personal preference use positive "TestX" style.